packages/client/modules/teamDashboard/components/TeamArchive/TeamArchive.tsx

```diff
@@ -152,7 +180,8 @@ const TeamArchive = (props: Props) => {
     oldEdgesRef.current = edges
   }, [edges, oldEdgesRef])
 
-  useDocumentTitle(`Team Archive | ${teamName}`, 'Archive')
+  showHeader && useDocumentTitle(`Team Archive | ${teamName}`, 'Archive')
```

Comment:
this may cause problems in the future because if showHeader ever changes, it'll cause a different number of hooks to run. may be cleaner to put this hook in the parent so there isn't an implicit contract that the `showHeader` prop cannot be modified.